src/test/java/com/pmarko09/medical_clinic_proxy/mapper/VisitMapperTest.java

```diff
@@ -0,0 +1,35 @@
+package com.pmarko09.medical_clinic_proxy.mapper;
+
+import com.pmarko09.medical_clinic_proxy.model.dto.AvailableVisitDto;
+import com.pmarko09.medical_clinic_proxy.model.dto.VisitDto;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class VisitMapperTest {
+
+    VisitMapper visitMapper = Mappers.getMapper(VisitMapper.class);
+
+    @Test
+    void mapToAvailableVisitDto() {
+        VisitDto visitDto = new VisitDto(
+                1L,
```

Comment:
tutaj też nie wiem czy jest potrzebne takie formatowanie

Reply:
imo lepsze takie przy LocaDateTime, bo przejrzysciej to widac